addons_rksv/pos_rksv/static/src/js/screens.js

```diff
@@ -206,7 +211,17 @@ odoo.define('pos_rksv.screens', function (require) {
             this.events['click .export_crypt'] = 'export_crypt';
             this.events['click .start_receipt_set_valid'] = 'start_receipt_set_valid';
         },
+        willStart: function() {
```

Comment:
I found it in other Odoo specific JS objects...is the willStart called before the start? Can you give me a pointer in the framework? to which framework does it belong?